easybuild/easyconfigs/p/plotutils/plotutils-2.6-GCC-11.3.0.eb

```
@@ -0,0 +1,60 @@
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'plotutils'
+version = '2.6'
+
+homepage = 'https://www.gnu.org/software/plotutils/'
+description = """The GNU plotutils package contains software for both programmers and 
+technical users. Its centerpiece is libplot, a powerful C/C++ function library for 
+exporting 2-D vector graphics in many file formats, both vector and bitmap. On the 
+X Window System, it can also do 2-D vector graphics animations.
+
+libplot is device-independent, in the sense that its API (application programming interface) 
+does not depend on the type of graphics file to be exported. A Postscript-like API 
+is used both for file export and for graphics animations. A libplot programmer needs 
+to learn only one API: not the details of many graphics file formats."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
```

Since `plotutils` should be used a dependency for https://github.com/easybuilders/easybuild-easyconfigs/pull/15950 I think it should also be in the `GCCcore` toolchain
```suggestion
toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
```

---

Good idea unfortunately `gawk` is `GCC` and not `GCCcore` which is one of the problems:
`gawk-5.1.1-GCC-11.3.0.eb`

---

Yes, but `plotutils` is currently in `GCCcore`: https://github.com/easybuilders/easybuild-easyconfigs/pull/15950
So this will cause problems.
Either we need to move `plotutils` to `GCC` or `gawk` to `GCCcore`.

---

changed back to `GCC`, since `gawk` dependency lives at `GCC` (see #17454)